### ISSUE TITLE
val_loss is inf and map decrease

### ISSUE THREAD

**Issue by cl886699** [ultralytics/ultralytics#403]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

I finetuning on dotav2 datasets from coco weights, yolov5 did not meet this problem.
`yolo task=detect mode=train model=yolov8l.pt data=./yolo/data/datasets/dotav2.yaml device=\'4,5,6,7\' batch=64 v5loader=True`
![yolov8_bug](https://user-images.githubusercontent.com/31495593/212651603-6f034516-4f29-4eda-9f17-cb06135d3d21.png)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by pderrenger** [comment]
@cl886699 hey there! 👋 Thanks for reaching out.

It looks like you're encountering an issue with the validation loss becoming infinite and a decrease in mAP while fine-tuning on the Dotav2 dataset using YOLOv8. This problem didn't occur when using YOLOv5.

To investigate this, I'd recommend checking the training configuration, data pre-processing, and other related parameters to ensure they are set up correctly for YOLOv8. Additionally, you might want to consider adjusting the learning rate, batch size, or conducting data analysis to identify any inconsistencies in the Dotav2 dataset.

Feel free to provide more details about your training setup or any specific configuration settings, and I'll be happy to assist you further in troubleshooting this issue with YOLOv8. 😊
